js/main.js

```diff
@@ -0,0 +1,24 @@
+// Функция, которая выводит случайное число в диапозоне от min до max
+function selfRandom(min, max)
+{
```

Comment:
```suggestion
function selfRandom(min, max) {
```
не нужно переносить скобку